src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
The return type of the `get` method should be parametrized with `Bulldozer`, not the raw `Machine` type to avoid using raw types and to ensure type safety.